src/magister.js

```diff
@@ -76,6 +77,11 @@ class Magister {
 		 * @readonly
 		 */
 		this.profileInfo = null
+		/**
+		 * @type {String}
+		 * @readonly
+		 */
+		this.token = null
```

Comment:
Is this needed since it's also set on the http object?
If it's useful to expose this for users, which I can definitely see, it might be worth to change this into a get property beneath the constructor, like so:
```javascript
/**
 * @type {String}
 * @readonly
 */
get token() {
    return this.http._token
}
```